cpp/models/ode_seir/model.h

```diff
@@ -72,7 +74,7 @@ class Model : public CompartmentalModel<InfectionState, Populations<InfectionSta
     *@param y The TimeSeries obtained from the Model Simulation.
     *@returns The computed reproduction number at the provided index time.
     */
-    IOResult<double> get_reproduction_number(size_t t_idx, const mio::TimeSeries<ScalarType>& y) noexcept
+    IOResult<ScalarType> get_reproduction_number(size_t t_idx, const mio::TimeSeries<ScalarType>& y) noexcept
```

Comment:
```suggestion
    IOResult<ScalarType> get_reproduction_number(size_t t_idx, const mio::TimeSeries<ScalarType>& y) 
```

remove according to Daniel's comment